### PR TITLE
483: modal

### PR DESCRIPTION
**Pull request by sutt0n**
image
image

### CODE REVIEW

File: web-client/src/presenter/actions/caseConsolidation/validateFileDocumentAction.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { isEmpty } from 'lodash';
+import { state } from 'cerebral';
+
+/**
+ * check to see if we can consolidate cases
```
Comment (matthopson) [inline web-client/src/presenter/actions/caseConsolidation/validateFileDocumentAction.js]: Consolidate? Also, should this be in the caseConsolidation folder?

File: web-client/src/presenter/actions/setModalErrorAction.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * sets modal error state
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.props the cerebral props
+ * @param {object} providers.store the cerebral store
+ * @returns {void}
+ */
+export const setModalErrorAction = ({ props, store }) => {
```
Comment (matthopson) [inline web-client/src/presenter/actions/setModalErrorAction.js:11]: Very surprised we don't already have something doing this.

File: web-client/src/presenter/sequences/fileDocumentToConsolidateCasesSequence.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { gotoFileDocumentSequence } from './gotoFileDocumentSequence';
+import { setModalErrorAction } from '../actions/setModalErrorAction';
+import { validateFileDocumentAction } from '../actions/caseConsolidation/validateFileDocumentAction';
+
+export const fileDocumentToConsolidateCasesSequence = [
```
Comment (matthopson) [inline web-client/src/presenter/sequences/fileDocumentToConsolidateCasesSequence.js]: This should probably be `..ExternalDocument...` I think.

Comment (matthopson) [reply web-client/src/presenter/sequences/fileDocumentToConsolidateCasesSequence.js]: Or something similar to `submitExternalDocumentSequence` ?

File: web-client/src/presenter/actions/setModalErrorAction.js (hunk 2 above)
Comment (sutt0n) [reply web-client/src/presenter/actions/setModalErrorAction.js:11]: We do, but their names are contextual...

Comment (sutt0n) [reply web-client/src/presenter/actions/setModalErrorAction.js:11]: It's ~probably~ something we'll have to refactor, tbh.

File: web-client/src/presenter/actions/caseConsolidation/validateFileDocumentAction.js (hunk 1 above)
Comment (sutt0n) [reply web-client/src/presenter/actions/caseConsolidation/validateFileDocumentAction.js]: Good point.

File: web-client/src/presenter/sequences/fileDocumentToConsolidateCasesSequence.js (hunk 3 above)
Comment (sutt0n) [reply web-client/src/presenter/sequences/fileDocumentToConsolidateCasesSequence.js]: Yes.

**Review by matthopson — APPROVED** [review]

**Pull request merged**